lib/convert.js

```diff
@@ -165,25 +177,81 @@ var sdk = require('postman-collection'),
           data: this.generatedStore.collection.toJSON()
         }]
       });
+    },
+
+    getOptions: function() {
+      return [
+        {
+          name: 'Set request name source',
+          id: 'requestNameSource',
+          type: 'string',
+          default: 'fallback',
+          description: 'Option for setting source for a request name'
+        },
+        {
+          name: 'Toggle for faking schema',
+          id: 'schemaFaker',
+          type: 'boolean',
+          default: true,
+          description: 'Option for Fake the schema using json or xml schema faker'
```

Comment:
Option for faking
Why the uppercase?
json and xml need to be uppercase, not lowercase